DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,19 @@
    "metadata": {},
```

Comment:
<p>The actual value of the heat capacity used in ACCESS-OM2 is <code>Cp = 3992.10322329649</code>.  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>